### PR TITLE
Changes related to Drupal 9 compatibility

### PR DESCRIPTION
**Pull request by andriyun**
During the work on the upgrade, https://github.com/OS2Forms/OS2Forms8 to Drupal9 was there a need to update some modules to their higher version.

This PR includes changes related to Drupal 9 compatibility

The effect of these changes can be tested on OS2Forms8 solution by using code from d9 brunch https://github.com/OS2Forms/os2forms8/tree/d9

It's added to OS2Forms8 via replacing of version os2forms_forloeb on the line https://github.com/OS2Forms/os2forms8/blob/d9/composer.json#L117

It is supposed to be changed to a new release tag afterward

### CODE REVIEW

**Comment by madsnorgaard** [comment]
I am happy with these changes as long as we merge to `develop` before `master`.

File: composer.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
     "cweagans/composer-patches": "^1.6.5",
     "drupal/admin_toolbar": "^3.0",
     "drupal/chosen": "2.10.0",
-    "drupal/ckeditor_a11ychecker": "1.0.0",
+    "drupal/ckeditor_a11ychecker": "1.x-dev#a2099d09f362d8b005a53e8f2845858424cac3da",
```
Comment (madsnorgaard) [inline composer.json:22]: Why do we not use `2.0.0-alpha1` here?

File: composer.json [hunk 2]
```diff
@@ -39,10 +39,10 @@
     "drupal/r4032login": "^2.1",
     "drupal/simple_ldap": "1.0.0-alpha3",
     "drupal/simplesamlphp_auth": "3.2.0",
-    "drupal/switch_page_theme": "1.0",
+    "drupal/switch_page_theme": "1.x-dev#ac38137f43cc72a8efba04bc127b87fbf2c28e06",
```
Comment (madsnorgaard) [inline composer.json:42]: This i would prefer to refactor as the use case is no longer. Seems unsupported too.

Comment (andriyun) [reply composer.json:42]: I don't know that use case for this module.
What do you mean by "refactor"?

File: composer.json (hunk 1 above)
Comment (andriyun) [reply composer.json]: Good catch. I didn't notice that there is a release for 2.x version. Updated!

**Comment by andriyun** [comment]
Hi @madsnorgaard 
I've changed the destination branch for PR from `master` to `develop`.

After this change, I see that PR got more commits to merge.
It looks like `develop` branch has not gotten all commits that the `master` branch has.
See more here https://github.com/OS2Forms/os2forms_forloeb/compare/develop...master

Is it something you can manage to solve on your side?

File: composer.json (hunk 2 above)
Comment (madsnorgaard) [reply composer.json:42]: It had a very specific use case revolving around being able to switch frontend theme per url's.

Here you can ignore my comment and we have noted that we have to remove this dependency.

**Comment by madsnorgaard** [comment]
Something has definitely gone wrong in either the release procedure or when merging feature branches. Perhaps we should just merge this PR as all changes have been reviewed already.

**Comment by andriyun** [comment]
Hi @madsnorgaard 
I wasn't sure that your last comment means that you agree on changes in PR. That is why I asked you for review one more time :)

**Review by madsnorgaard — APPROVED** [review]

**Pull request merged**